server/model/config.go

```diff
@@ -1309,20 +1310,36 @@ func (s *LogSettings) SetDefaults() {
 		s.FileJson = NewBool(true)
 	}
 
+	if s.AdvancedLoggingJSON == nil {
+		s.AdvancedLoggingJSON = []byte("{}")
+	}
+
 	if s.AdvancedLoggingConfig == nil {
 		s.AdvancedLoggingConfig = NewString("")
 	}
```

Comment:
We can remove this. The field is deprecated now, so we won't be assigning an empty string. New installs will just skip printing this in config.json if the field is null.